app/components/satis/dropdown/component.rb

```diff
@@ -64,12 +64,16 @@ def options_array(obj)
       end
 
       def option_value(item)
```

Comment:
text and value methods are currently being used by dropdown using collections only.
It becomes a bit tricky when we want to use this here... so i reverted the code for now.